app/attributes/attributecontroller.cpp

```diff
@@ -1511,6 +1511,11 @@ void AttributeController::renamePhotos()
         const QString src = InputUtils::getAbsolutePath( mFeatureLayerPair.feature().attribute( item->fieldIndex() ).toString(), prefix );
         QFileInfo fi( src );
         QString newName = QStringLiteral( "%1.%2" ).arg( val.toString() ).arg( fi.completeSuffix() );
+
+        // Remove leading slashes from newName following issue #3415
+        static const QRegularExpression leadingSlashes( "^/+" );
```

Comment:
Can we not make this static? If you'd like to make it static, make it a member of this class instead.

```suggestion
       const QRegularExpression leadingSlashes( "^/+" );
```